### ISSUE TITLE
Feature: Force elevation in new window

### ISSUE THREAD

**Issue by gerardog** [gerardog/gsudo#97]
One way to ensure UIPI isolation is to elevate in a new window. If a user wants to ensure UIPI is respected is to force all elevations in new windows. To achieve this the user could do `gsudo config ForceNewWindow true`, and then all subsequent elevations would been done in new windows.

It is desired that a batch/script that uses gsudo would produce the same outcome (but without breaking uipi) even if `ForceNewWindow` is true or false. Unfortunatelly 'gsudo' currently doesn't know if it was called from a script or an interactive shell, and I doubt it could or should tell the difference.

### Problems:

* Output capturing: would break
* Fast commands may open a new window, then close immediately. For example `gsudo -n dir` or `gsudo cmd /k dir` 
* Previous item could be fixed by opening cmd /k {command} instead of cmd /c (i.e. ´gsudo -n cmd /k dir´) but that would then break (change the outcome) of scripts using gsudo

**Comment by gerardog** [comment]
* First commit makes just ForceNewWindow equivalent to adding -n everywhere. 
Elevated commands fade away quite rapidly.
https://ci.appveyor.com/project/gerardog/gsudo/builds/40244333/artifacts

* Then made the new window shell stay open in d9e7c8f3
https://ci.appveyor.com/project/gerardog/gsudo/builds/40253541/artifacts

I have trouble finding an spec for this that fits all. Feedback wanted. Also, comparing with `gsudo config SecurityEnforceUacIsolation true`, is usefull. IMHO that one is easier to understand and does not break scripts.

**Comment by gerardog** [comment]
Completed in #190 

### Features:
- Added setting `gsudo config NewWindow.Force` to force every elevation in new window. Same as using `-n` every time.
- Added `--NoExit`: Keep elevated shell open after running {command}.
- Added `--NoClose`: When in new console, ask for keypress before closing the console.
- Added setting `gsudo config NewWindow.CloseBehaviour`: `OsDefault`, `KeepShellOpen` (--NoExit), `PressKeyToClose` (--NoClose)

- Config syntax more relaxed now:
```
gsudo.exe config loglevel debug
gsudo.exe config loglevel=debug
gsudo.exe config loglevel="debug"
gsudo.exe config loglevel =  debug
```


![image](https://user-images.githubusercontent.com/3901474/197410528-191aa2ce-e6a4-4619-b4c2-a045f3f41523.png)

![image](https://user-images.githubusercontent.com/3901474/197410600-e11da667-cb1e-4b11-927a-9ede8e03c31d.png)

![image](https://user-images.githubusercontent.com/3901474/197410476-bd7f2622-a076-4576-b2e0-7a5f77330b9b.png)

**Comment by gerardog** [comment]
Released in v2.0.0 => https://gerardog.github.io/gsudo/docs/tips/elevation-in-new-window
